doc_source/guide_waiters.rst

```diff
@@ -19,7 +19,12 @@ Waiters in the |sdk-php| Version 3
 Waiters help make it easier to work with *eventually consistent* systems by
 providing an abstracted way to wait until a resource enters into a particular
 state by polling the resource. You can find a list of the waiters supported by
-a client by viewing the API documentation of a service client.
+a client by viewing the API documentation for a single version of a service client.
+To navigate there, go to the client's page in our documentation and click on the

```

Comment:
Should this specify "API documentation" to distinguish from the dev guide documentation? Also, should "API documentation" itself be a link to the API docs homepage? https://docs.aws.amazon.com/aws-sdk-php/v3/api/index.html

Reply:
I updated the language to more clearly reflect which docs i was referring to and added the link.  Thank you